rust/core-lib/src/view.rs

```diff
@@ -1082,18 +1082,17 @@ impl View {
         }
     }
 
+    //get the line range of a selected region
```

Comment:
Nit: Rust doc comments begin with three `/`; I'd also add a space after the last `/`, and use a full sentence (that is, Capitalize the first word and include a terminal `.`)

Reply:
Thanks for pointing this out. Fixed.